src/logic/azext_logic/manual/custom.py

```diff
@@ -3,7 +3,43 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
+# pylint: disable=too-many-lines, unused-argument, protected-access
+
 from knack.util import CLIError
+from azure.cli.core.aaz import has_value
+from azext_logic.aaz.latest.logic.integration_account.map import Create as _MapCreate
+from azext_logic.aaz.latest.logic.integration_account.map import Update as _MapUpdate
+
+
+def logic_workflow_create(cmd, client,
+                          resource_group_name,
+                          name,
+                          definition,
+                          location,
+                          tags=None,
+                          state=None,
+                          endpoints_configuration=None,
+                          access_control=None,
+                          integration_account=None,
+                          integration_service_environment=None):
+    if 'definition' not in definition:
+        raise CLIError(str(definition) +
+                       " does not contain a 'definition' key")
+    workflow = {
+        'location': location,
+        'tags': tags,
+        'state': state,
+        'endpoints_configuration': endpoints_configuration,
+        'access_control': definition.get(
+            'accessControl', access_control),
+        'integration_account': integration_account,
+        'integration_service_environment': integration_service_environment,
+        'definition': definition['definition'],
+        'parameters': definition.get('parameters', None)
+    }
+    return client.create_or_update(resource_group_name=resource_group_name,
+                                   workflow_name=name,
+                                   workflow=workflow)
```

Comment:
What's these function used for?

Reply:
After updating the SDK, the `create_or_update` parameter was changed, `az logic workflow create` use it